app/src/main/AndroidManifest.xml

```diff
@@ -7,9 +7,9 @@
         android:allowBackup="true"
         android:dataExtractionRules="@xml/data_extraction_rules"
         android:fullBackupContent="@xml/backup_rules"
-        android:icon="@mipmap/ic_launcher"
+        android:icon="@drawable/picover_logo"
```

Comment:
Did You consider using adaptive icon ?

Reply:
I will consider it :D

Reply:
I decided to go this way, thanks for the advice, and DONE :D